### DISCUSSION TITLE
editorcmd launch successfully but cannot save

### DISCUSSION THREAD

**Discussion by shth** [tridactyl/tridactyl#4435] [Q&A]
1. download `vimr`
2. copied the vimr cli tool to `/usr/local/bin/vimr`
3. `set editorcmd /usr/local/bin/vimr` in tridactyl
4. focus an input field and hit <C-i>
5. vimr is launched successfully
6. type `:wq`
7. vimr show error `no filename`

Have I missed something? Any help is appreciated :pray:

ENV:
- tridactyl version 1.22.1
- `! /usr/local/bin/vimr` works fine
- `! vimr` produce `/bin/sh: vimr: command not found `

**Comment by bovine3dom** [comment]
Sorry for the delay, I only just saw this :)

I think you just need to use `:set editorcmd /usr/local/bin/vimr --wait` instead. If that doesn't work the one built into Tridactyl uses https://github.com/tridactyl/tridactyl/blob/7d819fa1401d5192dd99feb2942c35e743589f60/src/lib/native.ts#L126 but I'm not sure what the other arguments do.
